openreview/openreview.py

```diff
@@ -207,6 +208,45 @@ def get_profiles(self, email_or_id_list):
             return { p['email'] : Note.from_json(p['profile'])
                             for p in response.json()['profiles'] }
 
+    def get_pdf(self, id):
+        '''
+        Returns the binary content of a pdf using the provided note id
+        If the pdf is not found then this returns an error message with "status":404
+
+        Example Usage:
+
+        >>> f = get_pdf(id='Place Note-ID here')
+        >>> with open('output.pdf','wb') as op: op.write(f)
+        
+        '''
+        params = {}
+        params['id'] = id
+
+        headers = self.headers.copy()
+        headers['content-type'] = 'application/pdf'
+
+        response = requests.get(self.pdf_url, params = params, headers = headers)
+
+        return response.content
```

Comment:
I think you need to handle the response here, like the other calls, to keep consistency.